DungeonSource/Monster.java

```diff
@@ -26,26 +27,27 @@ public Monster(String name, int hitPoints, int attackSpeed,
 	this.maxHeal = maxHeal;
 	this.minHeal = minHeal;
 
-  }//end monster construcotr
+  }//end monster constructor
+  
+  public int getMinHeal()
```

Comment:
I added getters for these properties so the HealBehaviors can have access to them when calculating healing.